website/blog/2020-09-19-lithium.md

```diff
@@ -0,0 +1,266 @@
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Scala 0.26.0 and 0.27.0-RC1 support.
+- Analyze stacktrace command
+- Customizable package exclusions
+
+## Analyze stacktrace command
+
+Thanks to [kpbochenek](https://github.com/kpbochenek) Metals can now help with
+navigating stack traces using the new `metals.analyze-stacktrace` command. The
+only thing it needs is the actual stack trace string to be input as a parameter
+to the command. This new functionality works differently in Visual Studio Code
+and other supported editors.
+
+- for Visual Studio Code
+
+The only thing users need to do is copy the stack trace and run the
+`Analyze Stacktrace` command from the command palette. This will display a new
+web view, where it will be possible for users to navigate the files that are
+included in the exception.
+
+![stacktrace-vscode](https://i.imgur.com/WBU4hvT.gif)
+
+- for other editors
+
+To make it work with any other editors users need to run the command manually
+with the stacktrace as input. The command will generate a new file inside
+`.metals/stacktrace.scala` which can be used to navigate throught the stacktrace
+using code lenses.
+
+## Customizable package exclusions
+
+Usually, a number of packages are almost never used by the developers and could
+clutter the language server output while being of little benefit to the users.
+Metals would exclude those packages from indexing causing them to not be
+recommended for completions, symbol searches, and code actions. Previously that
+list was set in stone and consisted of a number of prefixes:
+
+```scala
+val defaultExclusions: List[String] = List(
+    "META-INF/", "images/", "toolbarButtonGraphics/", "jdk/", "sun/", "javax/",
+    "oracle/", "java/awt/desktop/", "org/jcp/", "org/omg/", "org/graalvm/",
+    "com/oracle/", "com/sun/", "com/apple/", "apple/"
+  )
+```
+
+This list might work in most cases, but users might also want to customize the
+list according to their preferences and domain they are working on. Starting
+with this release it is now possible to define additional exclusions as well as
+remove some from the default list. This can be done via additional Metals user
+setting:
+
+```json
+"metals.excluded-packages" : [
+  "akka.actor.typed.javadsl", "--javax"
+]
+```
+
+`--` might be used to remove some of the defaults if they are needed for your
+project. Thanks to [ckipp01](https://github.com/ckipp01) for contributing this
+feature!
+
+## Miscellaneous
+
+- New recommended Maven Scala template.
+- Fix go to source and auto imports in standalone worksheet files.
+- Fix missing classes from \_empty package in tree view.
+- Add a command to quickly create a new worksheet with a default name.
+
```

Comment:
So this is just a thought. Take it or leave it if you don't feel like it fits, but I've talked to a few people lately that are always surprised at the amount of work that goes into other projects to make what we do with Metals work. For example, in the last month I know a _significant_ amount of work has gone into Scalameta from @kpbochenek and I know you sent in quite a few fixes for mdoc @tgodzik. What do you think about maybe just mentioning under Miscellaneous something along the lines of:

> As you may know, Metals relies on various other Scalameta projects like [mdoc](https://scalameta.org/mdoc/), which powers worksheets and [Scalameta](https://scalameta.org/), which powers all SemanticDB powered features. There has been some great work done in those projects this last release to unlock and improve Dotty support in Metals. So special thanks to those projects and contributors.

Or something like that? What do you all think?

Reply:
I think it's a good idea,  Though not sure exactly where to put it :thinking:

Reply:
I put it to the section here, but changed the working a bit. Let me know what you think!

Reply:
👍 looks good!